source/blog/2019-02-16-community-survey-preview.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Preview Weekend: 2019 Ember Community Survey"
+author: Joseph Temple, Matthew Beale
+tags: Recent Posts, 2019
+responsive: true
+---
+ 
+We are thrilled to announce that we will be adding to the spring excitement
+surrounding [Ember Octane](https://github.com/emberjs/rfcs/blob/26c4d83fb66568e1087a05818fb39a307ebf8da8/text/0000-roadmap-2018.md)
+and [EmberConf 2019](https://emberconf.com/) with the release of the fifth
+annual Ember Community Survey, and as always, we want your input before we go
+live!
+
+In 2015, [201 Created](https://www.201-created.com/) began an annual survey of
+the Ember community. Over the past four years, data from the survey has helped
+us measure progress on issues as wide ranging as diversity and browser support.
+In 2017, free text responses were permitted for the first time giving community
+participants a way to provide feedback to the project and project leadership a
+new way to hear their voices. As 201 Created shuts down, it feels like a good
+time to promote the annual survey to an official project of the Ember Learning
+Team. Thank you 201 Created team (and interns who worked on this) for our first
+four years!
+
+In 2019 the Ember community is more vibrant than ever, with over 3000 members
+having joined the official Discord server (join via the [Ember Community page](https://www.emberjs.com/community/))
+that launched last September. We're eager to hear how people are using Ember
+and what they plan for the future.
+
+We'd like to get as much feedback as possible about the content of the survey
+before we go live. This year we're making an effort to move much of the survey
+process to the [ember-learn/annual-community-survey GitHub repo](https://github.com/ember-learn/annual-community-survey).
+Please review the [the list of 2019 survey questions](https://github.com/ember-learn/annual-community-survey/blob/master/2019/survey-questions.md)
```

Comment:
Move this up to the top if you want maximum participation. Keep it here if you only want a small # of really dedicated people to chime in.

Reply:
This is a good nudge. I try to follow the rule of "put the actionable thing first" myself, but you caught me forgetting it here. 👍